packages/fabrix/tests/render.tsx

```diff
@@ -17,10 +21,14 @@ export const testWithUnmount = async (
     ),
   });
 
-  await test();
```

Comment:
Without surrounding here with try-catch, `unmount()` would never be called when the previous tests fail.